client/landing/site.landing/coffee/main.coffee

```diff
@@ -37,7 +37,7 @@ do ->
     new MainController group
 
   kd.config             or= {}
-  kd.config.environment   = if location.hostname is 'koding.com' then 'production' else 'development'
+  kd.config.environment   = window._runtimeOptions.environment
```

Comment:
@sinan Is it possible to have landing loaded without `window._runtimeOptions` set in the page?